components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LoggingResource.java

```diff
@@ -18,21 +18,31 @@
 
 package org.wso2.micro.integrator.management.apis;
 
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.configuration.PropertiesConfigurationLayout;
```

Comment:
let's add this to the bundle imports

Reply:
Fixed in 8246bdbf3a22cb31344a32d01f94c055ab08f056